src/pip/_internal/req/req_install.py

```diff
@@ -63,6 +63,11 @@ def _strip_extras(path):
 
 
 class InstallRequirement(object):
+    """
+    Represents something that may be installed later on, may have information
+    about where to fetch the relavant requirement and also contains logic for
+    installing the said requirement.
+    """
```

Comment:
Hahahah. I was wondering where this change went.

I guess it's an improvement over status quo anyway here, so I'll let it stay unless someone has an issue with it.